src/main.ts

```diff
@@ -1350,8 +1350,17 @@ export class ProfilerFeature extends SqlOpsFeature<undefined> {
 			return undefined;
 		};
 
-		let disconnectSession = (sessionId: string): Thenable<boolean> => {
-			return undefined;
+		let disconnectSession = (ownerUri: string): Thenable<boolean> => {
+			let params: types.DisconnectSessionParams = {
+				ownerUri: ownerUri
+			};
+			return client.sendRequest(protocol.DisconnectSessionRequest.type, params).then(
+				r => true,
```

Comment:
probably better if this returns `r` instead of true, since the `boolean` return time implies that clients might want to send back false

Reply:
The type for these methods are Thenable<boolean>, that is why r => true is used here. I'd rather not change this otherwise i will have to make changes all over the places :(.

Reply:
why not change the request to be `Thenable<void>` then?

Reply:
I have to make a lot of changes event to change it to void. it is copying the sqlops typings file from the old repo, I have to change it pull from the new ADS repo as well as making changes in ADS, giving the fact this is not important enough, I will leave it as is.